Source/options.cpp

```diff
@@ -1221,6 +1221,7 @@ void OptionEntryLanguageCode::CheckLanguagesAreInitialized() const
 	languages.emplace_back("el", "Ελληνικά");
 	languages.emplace_back("en", "English");
 	languages.emplace_back("es", "Español");
+    languages.emplace_back("et", "Eesti");

```

Comment:
This should be indented with a tab rather than 4 spaces.

Reply:
Fixed! 🚀